base/est/CMakeLists.txt

```diff
@@ -0,0 +1,83 @@
+project(est NONE)
+
+javac(pki-est-classes
+    SOURCES
+        src/main/java/*.java
+    CLASSPATH
+        ${SERVLET_JAR}
+        ${JAXRS_API_JAR}
+        ${SLF4J_API_JAR}
+        ${COMMONS_CODEC_JAR}
+        ${JSS_JAR}
+        ${PKI_CMSUTIL_JAR}
+        ${PKI_CERTSRV_JAR}
+        ${PKI_CMS_JAR}
+    DEPENDS
+        pki-certsrv-jar pki-cms-jar
+    OUTPUT_DIR
+        ${CMAKE_CURRENT_BINARY_DIR}/classes
+)
+
+configure_file(
+    ${CMAKE_CURRENT_SOURCE_DIR}/src/main/resources/META-INF/MANIFEST.MF
+    ${CMAKE_CURRENT_BINARY_DIR}/MANIFEST.MF
+)
+
+jar(pki-est-jar
+    CREATE
+        ${CMAKE_BINARY_DIR}/dist/pki-est.jar
+    OPTIONS
+        m
+    PARAMS
+        ${CMAKE_CURRENT_BINARY_DIR}/MANIFEST.MF
+    INPUT_DIR
+        ${CMAKE_CURRENT_BINARY_DIR}/classes
+    DEPENDS
+        pki-est-classes
+)
+
+set(PKI_EST_JAR ${CMAKE_BINARY_DIR}/dist/pki-est.jar CACHE INTERNAL "pki-est.jar file")
+
+add_custom_target(pki-est-links ALL
+    COMMENT "Creating links for EST")
+
+add_custom_command(
+    TARGET pki-est-links
+    COMMAND ${CMAKE_COMMAND} -E make_directory webapp/lib
+    COMMAND ${CMAKE_COMMAND} -E create_symlink ${SLF4J_API_JAR} webapp/lib/slf4j-api.jar
+    COMMAND ${CMAKE_COMMAND} -E create_symlink ${SLF4J_JDK14_JAR} webapp/lib/slf4j-jdk14.jar
+    COMMAND ln -sf /usr/share/java/pki/pki-cms.jar ${CMAKE_CURRENT_BINARY_DIR}/webapp/lib/pki-cms.jar
+    COMMAND ln -sf /usr/share/java/pki/pki-cmsbundle.jar ${CMAKE_CURRENT_BINARY_DIR}/webapp/lib/pki-cmsbundle.jar
```

Comment:
The PR #4051 will merge pki-cms with pki-cmsbundle. Just a reminder!

Reply:
Thank you, I will rebase the PR very soon.